week-8/Homework/mandatory/3-dog-photo-gallery/main.js

```diff
@@ -0,0 +1,28 @@
+let root = document.getElementById("root");
```

Comment:
It is good that you are able to do this with JS. But I think you should try to leverage HTML and CSS to do some of this. For example, you can create the <button> manually in HTML and add any CSS for your elements in CSS file.

Reply:
Oh, ok I will do in the next project. Thank you!